.github/workflows/node.js.yml

```
@@ -57,3 +57,5 @@ jobs:
       - run: npm install
       - run: npm run lint
       - run: npm run test:unit
+        env:
+          JWT_SECRET: nb3ig2u9ng2vdvnuwebv
```

it would be better if you only upload the test file for .env to git (using .env.test)

---

Or use repository secrets:
```suggestion
        env:
          JWT_SECRET: {{ secrets.JWT_SECRET }}
```

---

The right syntax is `JWT_SECRET: ${{ secrets.JWT_SECRET }}`